### PR TITLE
index.html

### PR DESCRIPTION
**Pull request by Arinzemike1**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>My Portfolio</title>
```
Comment (Chigoz169) [inline index.html]: It's preferable to use your name or don't you think so?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+	<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css">
+	<link href="https://fonts.googleapis.com/css?family=Paytone+One&display=swap" rel="stylesheet">
+	<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js"></script>
```
Comment (Chigoz169) [inline index.html]: Styling isn't needed at this stage. 
So kindly do remove all forms of styling for now.

File: index.html (hunk 1 above)
Comment (Arinzemike1) [reply index.html]: This was already modified yesterday. Thanks though

File: index.html (hunk 2 above)
Comment (Arinzemike1) [reply index.html]: Okay, I will!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+	<div class="overall">
+        <header>
+            <nav id="nav-bar" class="navbar navbar-expand-lg navbar-dark navbar-inverse navbar-fixed-top">
+                <h4 class="navbar-brand" href="#">Arinze's <span>Design</span></h4>
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"  aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                        <ul class="navbar-nav ml-auto">
+                        <li class="nav-item"><a class="nav-link active-link" href="#">Home</a></li>
```
Comment (Chigoz169) [inline index.html:25]: Do ensure that you use correct indentations for the li tags.
Preferably make use of your code formatting tools.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css">
+	<link href="https://fonts.googleapis.com/css?family=Paytone+One&display=swap" rel="stylesheet">
+	<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js"></script>
+</head>
+<body data-spy="scroll" data-target=".navbar" data-offset="50">
+	<div class="overall">
+        <header>
+            <nav id="nav-bar" class="navbar navbar-expand-lg navbar-dark navbar-inverse navbar-fixed-top">
```
Comment (Chigoz169) [inline index.html:16]: From this point downwards to where the nav tag closes, kindly re-visit as i guess it is a bootstrap function and not indented as well.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+	<main>
+        <section id="homepage">
+            <div class="home-text">
+                <img src="picture.jpg" alt="personal picture">
+                <p class="name">Hi! I'm <span>Arinze I.C</span></p>
+                <p class="desc">Front-end web developer</p>
+                <a href="#">My Work</a>
+            </div>
+        </section>
+    </div>
```
Comment (Chigoz169) [inline index.html]: Where  is this tag attached to?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+        <section id="social-media">
+            <h3>Let's connect:</h3>
+            <div>
+                <a href="https://www.instagram.com/ari.n.ze/" target="_blank"><i class="fab fa-instagram"></i></a>
+                <a href="https://www.linkedin.com/in/arinze-ihemedu-14b738148/" target="_blank"><i class="fab fa-linkedin-in"></i></a>
+                <a href="https://www.quora.com/profile/Arinze-Ihemedu" target="_blank"><i class="fab fa-quora"></i></a>
+                <a href="https://github.com/Arinzemike1" target="_blank"><i class="fab fa-github"></i></a>
+            </div>	
+        </section>
+
```
Comment (Chigoz169) [inline index.html:107]: Why this line space? Do remove it if not in use.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+            <div class="container">
+                <h1>About Me</h1>
+                <div class="row">
+                    <div class="col-md-6 text-center about-col">
+                        <div class="profile">
+                            <img src="picpic.jpg" class="user">
+                            <p>Hi there! I'm Arinze Ihemedu, a graduate of applied informatics and system programming from Vitebsk State University. I've got some basic understanding of c++ programming language but here I am doing what I love most which is web development. My stack: HTML5, CSS3/Bootstrap, Javascript/Jquery.</p>
+                        </div>
+                    </div>
+
```
Comment (Chigoz169) [inline index.html]: Remove space

File: index.html [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+                        <p>GIT</p>
+                        <div class="progress">
+                            <div class="progress-bar" style="width: 30%;">50%</div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="contact-btn"><a href="#">Contact me</a></div>
+        </section>
+    </main>
```
Comment (Chigoz169) [inline index.html]: Indent well as it does not tally with it's opening tag.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
```
Comment (Chigoz169) [inline index.html]: Remove space

File: index.html [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="contact-btn"><a href="#">Contact me</a></div>
+        </section>
+    </main>
+	
+<footer class=" footer page-footer font-small blue pt-4">
+
```
Comment (Chigoz169) [inline index.html]: Remove

File: index.html [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+	<div class="container-fluid text-center text-md-left">
+	  <div class="row">
+		<div class="col-md-6 mt-md-0 mt-3 lg">
```
Comment (Chigoz169) [inline index.html]: I think i have been seing this for a while now. On this line of code, is this the proper indentation as compared to that on the preceeding line (line 98)? Do re-visit to verify and effect all amendments where necessary.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+		  <h5 class="text-uppercase">Arinze's <span>Design</span></h5>
+
```
Comment (Chigoz169) [inline index.html]: Remove

File: index.html [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+	<div class="container-fluid text-center text-md-left">
+	  <div class="row">
+		<div class="col-md-6 mt-md-0 mt-3 lg">
+		  <h5 class="text-uppercase">Arinze's <span>Design</span></h5>
+		
+		  <div class="footer-copyright text-left py-3">© 2020 Copyright
+			<strong>ARINZE'S DESIGN</strong>
+		  </div>
+		</div>
+
```
Comment (Chigoz169) [inline index.html]: Remove

File: index.html [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+		<div class="col-md-6 mt-md-0 mt-3 lg">
+		  <h5 class="text-uppercase">Arinze's <span>Design</span></h5>
+		
+		  <div class="footer-copyright text-left py-3">© 2020 Copyright
+			<strong>ARINZE'S DESIGN</strong>
+		  </div>
+		</div>
+  
+		<hr class="clearfix w-100 d-md-none pb-3">
+
```
Comment (Chigoz169) [inline index.html]: Remove

File: index.html [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+		
+		<div class="col-md-3 sos-med">
+			<div>
+				<a href="https://www.instagram.com/ari.n.ze/" target="_blank"><i class="fab fa-instagram"></i></a>
+				<a href="https://www.linkedin.com/in/arinze-ihemedu-14b738148/" target="_blank"><i class="fab fa-linkedin-in"></i></a>
+				<a href="https://www.quora.com/profile/Arinze-Ihemedu" target="_blank"><i class="fab fa-quora"></i></a>
+				<a href="https://github.com/Arinzemike1" target="_blank"><i class="fab fa-github"></i></a>
+			</div>
+		</div>
+
```
Comment (Chigoz169) [inline index.html]: Remove

File: index.html [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+		<div class="col-md-3 mb-md-0 mb-3">
+ 			<ul class="list-unstyled">
+				<li>
```
Comment (Chigoz169) [inline index.html]: Indentations please

File: README.md [hunk 17]
```diff
@@ -1,2 +1,15 @@
 # konga-portfolio
 My portfolio site
+
+<!--HEADER-->
+The header of this webpage contains the bootstrap nav and a text logo
```
Comment (chiamakaikeanyi) [inline README.md]: Can you remove bootstrap here as it's not meant to be used in the project?

File: index.html [hunk 18]
```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="keywords" content="Front-end web development">
```
Comment (chiamakaikeanyi) [inline index.html]: This should be comma-separated values. Also add other meta tags.

File: index.html [hunk 19]
```diff
@@ -0,0 +1,132 @@
...
+    <title>Arinze's Portfolio</title>
+    <link rel="shortcut icon" type="image/jpg" href="images/fav-icon.ico">
```
Comment (chiamakaikeanyi) [inline index.html]: The value of the type attribute and the value specified for the image type in the href attribute differ. 

Also note that JPG is not fully supported across browsers. Refer to the "How to use" and the "File format support" section of https://en.wikipedia.org/wiki/Favicon

File: index.html [hunk 20]
```diff
@@ -0,0 +1,132 @@
...
+</head>
+
+<body data-spy="scroll" data-target=".navbar" data-offset="50">
+    <div class="overall">
```
Comment (chiamakaikeanyi) [inline index.html:14]: Do you really need this?

File: index.html [hunk 21]
```diff
@@ -0,0 +1,132 @@
...
+        <header>
+            <nav id="nav-bar" class="navbar navbar-expand-lg navbar-dark navbar-inverse navbar-fixed-top">
+                <h4 class="navbar-brand" href="#">Arinze's <span>Design</span></h4>
```
Comment (chiamakaikeanyi) [inline index.html]: This should be a h1 tag which can be reduced in size later using CSS

File: index.html [hunk 22]
```diff
@@ -0,0 +1,132 @@
...
+                <button class="navbar-toggler" type="button" data-toggle="collapse"
+                    data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav ml-auto">
```
Comment (chiamakaikeanyi) [inline index.html:24]: Use descriptive class name here and in all other occurrences

File: index.html (hunk 3 above)
Comment (chiamakaikeanyi) [reply index.html:25]: Consider using prettier and enable format on save if you're making use of VSCode

File: index.html [hunk 23]
```diff
@@ -0,0 +1,132 @@
...
+                        <li class="nav-item"><a class="nav-link" href="#about">About Me</a></li>
+                        <li class="nav-item"><a class="nav-link" href="#social-media">Contact</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <main>
+            <section id="homepage">
+                <div class="home-text">
+                    <img src="images/picture.jpg" alt="personal picture">
```
Comment (chiamakaikeanyi) [inline index.html:36]: Let the alt text start with a capital letter and terminate with a dot.

File: index.html [hunk 24]
```diff
@@ -0,0 +1,132 @@
...
+                    <p class="name">Hi! I'm <span>Arinze I.C</span></p>
+                    <p class="desc">Front-end web developer</p>
+                    <a href="#">My Work</a>
```
Comment (chiamakaikeanyi) [inline index.html:39]: This should link to the work section

File: index.html [hunk 25]
```diff
@@ -0,0 +1,132 @@
...
+                </div>
+            </section>
+    </div>
+    <section id="social-media">
+        <h3>Let's connect:</h3>
+        <div>
+            <a href="https://www.instagram.com/ari.n.ze/" target="_blank"><i class="fab fa-instagram"></i></a>
```
Comment (chiamakaikeanyi) [inline index.html]: When you use target="_blank", always add rel="noopener" or rel="noreferrer". Visit https://developers.google.com/web/tools/lighthouse/audits/noopener for more information.

File: index.html [hunk 26]
```diff
@@ -0,0 +1,132 @@
...
+            <a href="https://www.linkedin.com/in/arinze-ihemedu-14b738148/" target="_blank"><i
+                    class="fab fa-linkedin-in"></i></a>
+            <a href="https://www.quora.com/profile/Arinze-Ihemedu" target="_blank"><i class="fab fa-quora"></i></a>
+            <a href="https://github.com/Arinzemike1" target="_blank"><i class="fab fa-github"></i></a>
+        </div>
+    </section>
+    <section id="about">
+        <div class="container">
+            <h1>About Me</h1>
```
Comment (chiamakaikeanyi) [inline index.html:57]: This shouldn't be a `h1` tag

File: index.html [hunk 27]
```diff
@@ -0,0 +1,132 @@
...
+            <div class="row">
+                <div class="col-md-6 text-center about-col">
+                    <div class="profile">
+                        <img src="images/picpic.jpg" class="user" alt="about image">
```
Comment (chiamakaikeanyi) [inline index.html]: Consider using a descriptive image name and alt text

File: index.html [hunk 28]
```diff
@@ -0,0 +1,132 @@
...
+                        <p>Hi there! I'm Arinze Ihemedu, a graduate of applied informatics and system programming from
+                            Vitebsk State University. I've got some basic understanding of c++ programming language but
+                            here I am doing what I love most which is web development. My stack: HTML5, CSS3/Bootstrap,
+                            Javascript/Jquery.</p>
+                    </div>
+                </div>
+                <div class="col-md-6 skills-bar">
+                    <p>HTML</p>
+                    <div class="progress">
```
Comment (chiamakaikeanyi) [inline index.html:70]: Use semantic html here and in all occurrences. Refer to https://www.w3schools.com/tags/tag_progress.asp

File: index.html [hunk 29]
```diff
@@ -0,0 +1,132 @@
...
+                        <div class="progress-bar" style="width: 70%;">70%</div>
+                    </div>
+                    <p>GIT</p>
+                    <div class="progress">
+                        <div class="progress-bar" style="width: 30%;">50%</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="contact-btn"><a href="#">Contact me</a></div>
```
Comment (chiamakaikeanyi) [inline index.html]: This section should contain a form that collects user's message eventually and sends to your mail. For now it should just be a form

File: index.html [hunk 30]
```diff
@@ -0,0 +1,132 @@
...
+    </section>
+    </main>
+    <footer class=" footer page-footer font-small blue pt-4">
```
Comment (chiamakaikeanyi) [inline index.html]: Are all these classes really necessary? You wouldn't use Bootstrap

File: index.html [hunk 31]
```diff
@@ -0,0 +1,132 @@
...
+                <div class="col-md-6 mt-md-0 mt-3 lg">
+                    <h5 class="text-uppercase">Arinze's <span>Design</span></h5>
+                    <div class="footer-copyright text-left py-3">© 2020 Copyright
+                        <strong>ARINZE'S DESIGN</strong>
+                    </div>
+                </div>
+                <hr class="clearfix w-100 d-md-none pb-3">
+                <div class="col-md-3 sos-med">
+                    <div>
+                        <a href="https://www.instagram.com/ari.n.ze/" target="_blank"><i
```
Comment (chiamakaikeanyi) [inline index.html:129]: When you use target="_blank", always add rel="noopener" or rel="noreferrer". Visit https://developers.google.com/web/tools/lighthouse/audits/noopener for more information.

**Pull request merged**